### ISSUE TITLE
Add trusted neighbour

### ISSUE THREAD

**Issue by RaduCStefanescu** [code4romania/next-door#26]
As an user in the application I need to be able to add a list of trusted neighbours.

We need to create a new entity that links two `Person` entities through the relationship `trusted neighbour`. For now, the entity should only encapsulate minimal information:
- id of the trustee
- id of the trusted Person
- timestamp of the relationship

**Comment by mischiedorian** [comment]
I would like to take this one. `Person` entity should be done before this?

**Comment by georgebejan** [comment]
@mischiedorian Yes. The person task https://github.com/code4romania/next-door/issues/23 is open right now, you can take it if you wish

**Comment by mischiedorian** [comment]
@georgebejan, when #23 will be closed, I will take this one
